classes/class3/cat_homework/routes/index.js

```diff
@@ -0,0 +1,70 @@
+var catModel = require('../models/catmodels');
+
+
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+function randInt(min, max) {
+	return Math.floor(Math.random() * (max - min)) + min;
+}
+
+var names = ["Tubs", "Samantha", "Speckles", "Sherlock", "Mr. Biggs", "Ella", "Marbles", "Delilah", "Zippy", "Sal", "Madeline", "Delaney", "Giles", "Mr.Mistoffelees","Mittens"];
+var colors = ["orange", "grey", "black", "white", "calico", "tabby", "silver", "tan", "gold", "blue", "purple", "red", "magenta", "yellow", "green", "lilac", "pink"];
+
+function Cat() {
+	this.age = randInt(0, 23);
+	this.name = names[randInt(0,names.length)];
+	this.color = [colors[randInt(0,colors.length)], colors[randInt(0,colors.length)], colors[randInt(0,colors.length)]];
+}
+
+var newCat = function(req,res) {
+	var myCat = new Cat();
```

Comment:
An interesting use of a javascript class. Nothing inherently wrong with it, but you will more often see something like this:
```node
var catObj = {
  age: randInt(0, 23),
  name: names[randInt(0,names.length)],
  color: [colors[randInt(0,colors.length)], colors[randInt(0,colors.length)], colors[randInt(0,colors.length)]]
}
var newCat = new catModel(catObj);
```